service/src/test/java/bio/terra/profile/pact/SamServiceTest.java

```diff
@@ -51,7 +50,7 @@ public RequestResponsePact userStatusPact(PactDslWithProvider builder) {
   }
 
   @Test
-  @PactTestFor(pactMethod = "statusApiPact")
+  @PactTestFor(pactMethod = "statusApiPact", pactVersion = PactSpecVersion.V3)
```

Comment:
Specifying `pactVersion` fixed the failures we were seeing on the dependabot PR.

Reply:
weird... I don't think that was in any of the docs, tutorials, stackexchange threads, or arcane tombs I read about pact.

Reply:
It's in the examples now….